src/Attribute.php

```diff
@@ -12,6 +12,8 @@
  *
  * Usually attributes are not instantiated directly, but created through an HTML
  * element's exposed methods.
+ *
+ * @phpstan-type AttributeValueFormat string|bool|string[]|null
```

Comment:
Why the `Format` suffix? It's a type after all.

```suggestion
 * @phpstan-type _AttributeScalar string|bool|null
 * @phpstan-type AttributeValue _AttributeScalar|array<_AttributeScalar>
```